project/AppDependencies.scala

```diff
@@ -6,24 +6,24 @@ object AppDependencies {
   val akkaVersion = "2.6.10"
 
   val compile = Seq(
-    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % "5.11.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"        % "0.54.0",
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % "5.14.0",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"        % "0.55.0",
```

Comment:
@AndyHWChung @meghana-viswanath  Funny thing we are already using this repo :)

Reply:
oh yeah :laughing: maybe it was brought in automatically at the start but we never noticed it